contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -150,6 +150,8 @@ origin point. For expected errors (`errors` that can happen during routine opera
 Use of Fatal should be discouraged and it's not available in new functions. Instead of depending on the logger to exit
 the process, you should call `os.Exit()` yourself.
 
+Fatal calls use a default exit code of 255. When migrating, please use an exit code of 1 and include an "ACTION REQUIRED:" release note.
```

Comment:
@ehashman 
i'm realizing that depending on how you split the work, we can have a lot of PRs that have action required.
ideally, we should have one per component.

also i think the release notes tool has the option to override release notes, so the release team can help there.

Reply:
Yeah I think the release team is gonna have to squash these all. It's not going to be possible to do these all in one PR.